src/main/java/frc/robot/RobotContainer.java

```diff
@@ -81,7 +88,7 @@ public RobotContainer()
             // Real robot, instantiate hardware IO implementations
             case REAL:
                 _drive = new Drive(
```

Comment:
Once the `Gyro` subsystem is create above this line, we'll need to add it as a parameter to the `Drive` subsystem's constructor:

```suggestion
                _drive = new Drive(_gyro,
```